tests/unit/components/ont/runs/ONTRunInstrumentFlowcells.spec.js

```diff
@@ -69,7 +70,7 @@ describe('ONTRunInstrumentFlowcells', () => {
     })
 
     it('must have instruments', () => {
-      const expected = mockInstruments.map((i) => {
+      const expected = Object.values(mockInstruments).map((i) => {
```

Comment:
Would this be better in the factory?

Reply:
Yes, it is used in a couple of places. But we need this as object as well in other places, so an additional method to return as array in factory might be good I hope.